### PR TITLE
Feature/create call notes model

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony ready for review cc @justinhacks

File: mentorship/mentor/views.py [hunk 1]
```diff
@@ -87,6 +87,15 @@ class CategoryViewSet(mixins.ListModelMixin, mixins.RetrieveModelMixin,
...
         return Category.objects.live()
 
+    @parser_classes((FormParser, MultiPartParser))
+    def image(self, request, *args, **kw):
+        category = self.get_object()
+        if not category.image:
+            raise NotFound(detail="Image not available")
+        return HttpResponse(
+            category.image.read(),
+            content_type="image/png")
```
Comment (justinhacks) [inline mentorship/mentor/views.py]: Could we add a test for this?

**Comment by justinhacks** [comment]
Left one comment, looks good.

File: mentorship/mentor/views.py (hunk 1 above)
Comment (codiebeulaine) [inline mentorship/mentor/views.py]: @justinhacks not sure how that got in there, that was me trying to fix the image thing on a different branch

**Comment by justinvdm** [comment]
:+1:, though maybe we should first get travis happy. Failure looks unrelated, though I'm not too sure what its about or how to fix it.

**Pull request merged**